loader/src/test/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/LoaderCacheSpec.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2018-2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.storage.bigquery
+package loader
+
+import java.time.Instant
+import cats.data.NonEmptyList
+import com.snowplowanalytics.iglu.client.ClientError
+import com.snowplowanalytics.iglu.client.resolver.LookupHistory
+import com.snowplowanalytics.iglu.core.{SchemaKey, SchemaVer}
+import com.snowplowanalytics.iglu.schemaddl.bigquery.{Field, Mode, Type}
+import com.snowplowanalytics.snowplow.badrows.FailureDetails
+
+import org.specs2.mutable.Specification
+
+class LoaderCacheSpec extends Specification {
+  import com.snowplowanalytics.snowplow.storage.bigquery.loader.LoaderCache._
+
+  sequential
+
+  "get should" >> {
+    "return None if the key is not in the cache" >> {
+      val __ = underlying.clear()
```

Comment:
```suggestion
      underlying.clear()
```
Is there a reason why you use `val _ = ` everywhere instead of just writing the expression without the `val` ?

Reply:
Just a preference. Helps me very easily find all the places where it happens.

Reply:
Would have switch `__` with single underscore though.

Reply:
Ah, that is already taken up :)

Reply:
Not sure what you mean be taken up. `_` is a throwaway, it cannot be taken up.

Reply:
I'm using it for another throwaway value in some of the tests, eg: https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/0.5.0-rc26/loader/src/test/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/LoaderCacheSpec.scala#L43

And I wanted to be consistent, so `clear` is always `__`.